python-avd/pyavd/_eos_designs/shared_utils/utils.py

```diff
@@ -301,3 +301,21 @@ def get_vrf(
                 return self.inband_mgmt_vrf or "default"
             case _:
                 return vrf_input
+
+    def _get_local_interface(self: SharedUtilsProtocol, input_interface: str | None) -> str | None:
```

Comment:
```suggestion
    def get_local_interface(self: SharedUtilsProtocol, input_interface: str | None) -> str | None:
```

and adjust calling code

Reply:
Fixed!